src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -324,4 +324,10 @@
   <data name="PX1007" xml:space="preserve">
     <value>NoXmlCommentForPublicEntityOrDacProperty</value>
   </data>
+  <data name="PX1019" xml:space="preserve">
```

Comment:
@k-s-popova  please review short names for diagnostics used in suppression

Reply:
@SENya1990 , I suggest to change AutoNumberingAttribute to AutoNumberAttribute in both short names just to have the names shorter.

Reply:
done